include/Dialects/LinalgTransform/LinalgTransformOps.td

```diff
@@ -33,6 +34,34 @@ class Linalg_Transform_Operation<string name, list<OpTrait> props = []>
 
 //===----------------------------------------------------------------------===//
 
+def ScopeOp : Linalg_Transform_Operation<"scope",
```

Comment:
Bikeshed: given that this operation is a utility and we don't expect to see it in the transform description, I would reflect that in the name. Something like `util.scope` would be better.

Reply:
Sure. Once this feature gets more use, I'm planning to move it into its own dialect (with only `scope` and `forward` as the operations).